kratos/elements/embedded_nodal_variable_calculation_element_simplex.cpp

```diff
@@ -102,10 +104,11 @@ void EmbeddedNodalVariableCalculationElementSimplex<array_1d<double,3>>::Calcula
     MatrixType &rLeftHandSideMatrix,
     ProcessInfo &rCurrentProcessInfo)
 {
-        // Check size
+    // Check size and initialize
     if (rLeftHandSideMatrix.size1() != 6 || rLeftHandSideMatrix.size2() != 6) {
         rLeftHandSideMatrix.resize(6, 6, false);
     }
+    rLeftHandSideMatrix = ZeroMatrix(6,6);
```

Comment:
This may be needed as the loop don't iterate all the components

Reply:
Actually this was the one giving problems. I only added the other ones to be consistent. I'll remove the ones that are not needed.

Reply:
this makes sense